programs/native/bpf_loader/src/lib.rs

```diff
@@ -52,11 +53,12 @@ pub fn helper_printf(arg1: u64, arg2: u64, arg3: u64, arg4: u64, arg5: u64) -> u
         + size_arg(arg5)
 }
 
-fn create_vm(prog: &[u8]) -> Result<rbpf::EbpfVmRaw, Error> {
-    let mut vm = rbpf::EbpfVmRaw::new(None)?;
+fn create_vm(prog: &[u8]) -> Result<EbpfVmRaw, Error> {
+    let mut vm = EbpfVmRaw::new(None)?;
     vm.set_verifier(bpf_verifier::check)?;
+    vm.set_max_instruction_count(36000)?;
```

Comment:
nit: justify 36000 in a comment.   `// 36000 seems like a lot  🤷‍♂️ , probably need to tune this value though...`